pkg/didcomm/crypto/jwe/authcrypt/authcrypt_test.go

```diff
@@ -122,32 +369,43 @@ func TestBadCreateCipher(t *testing.T) {
 }
 
 func TestRefEncrypt(t *testing.T) {
-	// reference from
+	// reference php crypto material similar to
 	// https://github.com/hyperledger/aries-rfcs/issues/133#issuecomment-518922447
-	var senderPriv = "6tsNPgZAg-NWM3s4S0VOOWM2yrcOfwsCrN0JGFEWaWw"
-	var senderPub = "QbvqozxGQ3U8FDLmlKOx8Hd5GiozMRO2pwrevZ5ZFTM"
-	var recipientPub = "-u0zk9iY_ZS2wP2z4zuLjR7_kz_kxVU0anRz8_A66T0"
-	var payload = []byte("SGVsbG8gV29ybGQh")
+	var senderPrivStr = "kPvYCto43QBAfpcChJf2mhfS8wb8Z6_tnPjgxFHYJyQ"
+	senderPriv, err := base64.RawURLEncoding.DecodeString(senderPrivStr)
+	require.NoError(t, err)
+	var senderPubStr = "2-N3lmOYaP9Sn7cLxg5KuyCD127EzGe1uJ-cue1YlVE"
+	senderPub, err := base64.RawURLEncoding.DecodeString(senderPubStr)
+	require.NoError(t, err)
+	var recipientPrivStr = "fyEbHm74hLCUAszVbr3YG4WR_f5QcxxDC2hLvZ9njZ0"
+	recipientPriv, err := base64.RawURLEncoding.DecodeString(recipientPrivStr)
+	require.NoError(t, err)
+	var recipientPubStr = "ZY-f6BVk_X5pQcX43f-aO_J9XE3uRGu6b5kr0nCCpnE"
+	recipientPub, err := base64.RawURLEncoding.DecodeString(recipientPubStr)
+	require.NoError(t, err)
+	var payloadStr = "SGVsbG8gV29ybGQh"
+	payload, err := base64.RawURLEncoding.DecodeString(payloadStr)
+	require.NoError(t, err)
 
 	//nolint:lll
 	const refJWE = `{
```

Comment:
How was the refJWE generated? Some documentation in the comments is desired.

Reply:
refJWE was generated by PHP.. I will add a comment..